File: app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentComponent.java
package org.dhis2.usescases.enrollment;

import org.dhis2.data.dagger.PerActivity;

import dagger.Subcomponent;

@PerActivity
@Subcomponent(modules = EnrollmentModule.class)
public interface EnrollmentComponent {
    void inject(EnrollmentActivity activity);
}

File: app/src/main/java/org/dhis2/usescases/enrollment/EnrollmentComponent.kt
package org.dhis2.usescases.enrollment

import dagger.Subcomponent
import org.dhis2.data.dagger.PerActivity

@PerActivity
@Subcomponent(modules = [EnrollmentModule::class])
interface EnrollmentComponent {
    fun inject(activity: EnrollmentActivity)
}